lib/prism/node_ext.rb

```diff
@@ -118,7 +118,7 @@ def full_name_parts
         current = current.parent
       end
 
-      unless current.is_a?(ConstantReadNode)
+      unless current.is_a?(ConstantReadNode) || current == nil
```

Comment:
```suggestion
      if !current.is_a?(ConstantReadNode) && !current.nil?
```

I never understand `unless` without significant headaches...